src/MailableFactory.php

```diff
@@ -20,7 +20,11 @@ public function __construct(ArgumentValueProvider $argumentValueProvider)
     public function getInstance(string $mailableClass, string $toEmail, $defaultValues): Mailable
     {
         if (! class_exists($mailableClass)) {
-            throw MailableDoesntExist::withClass($mailableClass);
+            $mailableClass = sprintf('%s\\%s', config('mailable-test.base_namespace'), $mailableClass);
```

Comment:
Is this necessary? We're already doing a `class_exists` in `SendTestMail`, can't we normalize the class name there so this class doesn't care about resolving the base namespace?